src/Player.cpp

```diff
@@ -136,23 +121,22 @@ void Player::handleTempBarricade(SDL_Renderer *renderer) {
         double angle = marine->getAngle();
         int distance = 100;
```

Comment:
should be const minimum and if it is the default distance it should be in the header as constexpr